app/controllers/ProfileController.scala

```diff
@@ -64,20 +65,22 @@ class ProfileController @Inject() (
 
   private def cleanseProfileData(answers: UserAnswers): Future[UserAnswers] =
     for {
-      updatedAnswersRemovedUkims       <-
+      updatedAnswersRemovedUkims            <-
```

Comment:
Not a you thing, don't worry about it now, but maybe this should use that journey cleanup thing

Reply:
I did also think this on my ticket, for now this fits the bill but I feel like cleanup logic should be centralised